packages/starknet-snap/test/src/index.test.ts

```diff
@@ -87,36 +84,6 @@ describe('onRpcRequest', function () {
       );
     }
   });
-
-  it('does not hide the error message if an error is thrown and `LogLevel` is not `OFF`', async function () {
```

Comment:
I guess this is handled in a new way in next PR ?

Reply:
yes